python-manual/modules/ROOT/pages/performance.adoc

```diff
@@ -0,0 +1,160 @@
+= Performance recommendations
+
+- *Specify the target database on all queries* with the `database` parameter when creating new xref:transactions.adoc[sessions]. If no database is provided, the driver has to send an extra request to the server to figure out what the default database is. The overhead is minimal for a single session, but becomes significant over hundreds of sessions.
++
+[source,python,role=noheader]
+----
+# Good practice
+driver.session(database="<DB NAME>")
+----
++
+[source,python,role=noheader]
+----
+# Bad practice
+driver.session()
+----
+
+// - Execute as many queries within the same session. Minimise the number of transactions?
+
+- *Use xref:query-simple#query-parameters[query parameters]* instead of hardcoding or concatenating values into queries. This allows the database to properly cache queries.
```

Comment:
```suggestion
- *Use xref:query-simple#query-parameters[query parameters]* instead of hardcoding or concatenating values into queries. 
This allows the database to properly cache queries.
```